MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -1665,7 +1714,14 @@ void EnggDiffractionPresenter::doFocusing(const EnggDiffCalibSettings &cs,
   MatrixWorkspace_sptr inWS;
 
   const std::string vanNo = m_view->currentVanadiumNo();
-  loadOrCalcVanadiumWorkspaces(vanNo, cs.m_inputDirCalib, vanIntegWS,
+
+  // Append instrument name if numerical only entry
+  std::string vanFileName;
```

Comment:
Not the point of this branch, but because of the way this interface has evolved we have some confusion between "No" and "Filename". `No` is the number. This vanFilename would be something like ENGINX22345 which is closer to the file name but still missing the 0s and the extension. Not totally sure, but maybe we could introduce here another name, "fileHint"? The smart file finder behind `Load`  will turn that into a file name as it is found on disk (and with a full path).